.github/install/installconsul.yaml

```diff
@@ -0,0 +1,13 @@
+name: user
+vars:
+  controlplaneNS: consul
+  consulVersion: 1.12.2
+  profile: default
+services:
+  consulservicemesh:
+    type: ConsulMesh
+    namespace:  $(#ref.vars.controlplaneNS)
+    settings:
+      version: $(#ref.vars.consulVersion)
```

Comment:
We recently had some changes in the spec. The version field now is a sibling of `type` and `namespace` so it will go one level up. No need to add settings now for ConsulMesh component.

Reply:
So I remove the entire `settings` section?